driver/bpf/fillers.h

```diff
@@ -6891,17 +6857,13 @@ FILLER(sched_prog_fork_3, false)
 	/* Parameter 21: pid_namespace init task start_time monotonic time in ns (type: PT_UINT64) */
 	if (pidns)
 	{
-	struct task_struct *child_reaper = (struct task_struct *)_READ(pidns->child_reaper);
-		if (child_reaper)
-		{
-			pidns_init_start_time = _READ(child_reaper->start_time);
-		}
+		struct task_struct *child_reaper = (struct task_struct *)_READ(pidns->child_reaper);
+		pidns_init_start_time = _READ(child_reaper->start_time);
 	}
 	res = bpf_val_to_ring_type(data, pidns_init_start_time, PT_UINT64);
-	if (res != PPM_SUCCESS)
-	{
-		return res;
-	}
+	CHECK_RES(res);
```

Comment:
minor: strictly not necessary due to the following `return res`, except being consistent to write+check for the previous calls.

Reply:
This file seems overall terribly malformatted, perhaps better to fix it in a cleanup PR.